target/linux/pistachio/config-4.4

```diff
@@ -577,34 +572,9 @@ CONFIG_USB_HID=y
 CONFIG_USB_LIBCOMPOSITE=y
 CONFIG_USB_MASS_STORAGE=y
 CONFIG_USB_MON=y
-CONFIG_USB_NET_AX8817X=y
-CONFIG_USB_NET_CDCETHER=y
-CONFIG_USB_NET_CDC_NCM=y
-CONFIG_USB_NET_DM9601=y
-CONFIG_USB_NET_DRIVERS=y
-CONFIG_USB_NET_MCS7830=y
-CONFIG_USB_NET_NET1080=y
-CONFIG_USB_NET_RNDIS_HOST=y
-CONFIG_USB_NET_SMSC75XX=y
-CONFIG_USB_NET_SMSC95XX=y
 CONFIG_USB_OTG=y
-CONFIG_USB_PEGASUS=y
-CONFIG_USB_RTL8150=y
-CONFIG_USB_RTL8152=y
-CONFIG_USB_SERIAL=y
-# CONFIG_USB_SERIAL_CONSOLE is not set
-CONFIG_USB_SERIAL_CP210X=y
-CONFIG_USB_SERIAL_FTDI_SIO=y
```

Comment:
The usb serial port at the top of the board runs directly through the FTDI chip. If you are getting a working serial console over USB without that driver then its not needed :)